casatables/src/lib.rs

```diff
@@ -529,6 +531,85 @@ where
     )
 }
 
+/// From `casacore::TableDesc`:
+///
+/// Define the structure of a Casacore table
+///
+/// A TableDesc object contains the description, or structure, of a table.
+/// This description is required for the creation of a new table.  
+/// Descriptions are subsequently associated with every table and
+/// embedded in them.
+///
+/// # Examples
+///
+/// Create a 
+///
+/// ```rust
+/// use rubbl_casatables::{GlueDataType, TableDesc};
+///
+/// let mut table_desc = TableDesc::new("TYPE");
+/// table_desc
+///     .add_scalar_column(GlueDataType::TpUInt, "column name")
+///     .unwrap();
+/// ```
+pub struct TableDesc {
+    handle: *mut glue::GlueTableDesc,
+    exc_info: glue::ExcInfo,
+}
+
+impl TableDesc {
+    /// Create a new TableDesc.
+    ///
+    /// `type` - effectively the name of the table. From casacore:
+    ///     This name can be seen as the table type in the same way as a
+    ///     class name is the data type of an object.
```

Comment:
Yeah, that casacore documentation is not very helpful. Is there a specific value that's used for MS data? If so, it would be good to tell people what it is.

Reply:
Hmmm, I'm not sure. @cjordan any ideas?

Reply:
See discussion in the main PR thread — I think we've figured out what's going on with this.